### DISCUSSION TITLE
Flexible bonding curves for creators

### DISCUSSION THREAD

**Discussion by 1R053** [deso-protocol/dips#191] [Large CIPs]
Creator coins may serve different purposes. Not all purposes are well supported with a quickly accelerating bonding curve.

Each creator should be able to decide, which bonding curve will be used for his account.

There could be various options:
- steady price: bonding curve increases logarithmically, great to have an almost identical price for most users
- linear price: bonding curve increases linearly, this allows a reasonable incentivation for early investors, while limiting rug pulling effects
- accelerating price: bonding curve follows a quadratic formula, great if mostly early investors should be supported

For each type there could be the option to set the start price and the curve steepness factor.

To allow a new creator to think a little about the effect, the bonding curve customisation could remain changeable, until the first coins have been minted. After the first coin for an account is minted, the bonding curve remains fixed.

**Comment by 0xNordian** [comment]
I usually do like the idea of flexibility and customization. However, don't you think this will make the experience on Deso more complex for the regular user?

**Comment by 1R053** [comment]
Yes, it does make the experience more complex. However, the current experience is imo not sustainable, especially due to the rug pulling issue.

In my opinion default should be a linear bonding curve in the future. The experience the last year has shown, that the quadratic curve simply does not work under normal circumstances. It only attracts users initially, but usually almost no users at all in the long term. Furthermore, it mainly attracts speculative users, not really the type supporting creators in the long term.

The parameters could be presented in a simple visual way to the creator upon signup,  the advanced features customisation can be optional. 

For users the curve can also be displayed visually quite easily as well. It is then not much more complex than the founder reward topic.
